src/components/ClientForm/ClientForm.tsx

```diff
@@ -0,0 +1,165 @@
+  const [goals, setGoals] = useState<string>();
+  const [coachGoals, setCoachGoals] = useState<string>();
+
+  function handleChange(val: string, id: string) {
+    switch (id) {
+      case "trainingFor":
+        setTrainingFor(val);
+        break;
+      case "futureDaysAWeek":
+        setFutureDaysWeek(val);
+        break;
+      case "currentDaysAWeek":
+        setCurrentDaysWeek(val);
+        break;
+      case "currentFitness":
+        setCurrentFitness(val);
+        break;
+      case "injuries":
+        setInjuries(val);
+        break;
+      case "favoriteExercise":
+        setFavorite(val);
+        break;
+      case "commonExercise":
+        setMostOften(val);
+        break;
+    }
+  }
+
+  const onSubmit = async (e: any) => {
+    e.preventDefault();
+    const passingArray = [
+      name,
+      phone,
+      email,
+      trainingFor,
+      currentDaysWeek,
+      futureDaysWeek,
+      currentFitness,
+      injuries,
+      yesInjuries,
+      favorite,
+      mostOften,
+      goals,
+      coachGoals,
+    ];
+    await sendContactForm(passingArray.toString());
+  };
+
+  return (
+    <form className={styles.main}>
+      <h2 className={styles.formHeader}>New Client Form</h2>
+      <div className={styles.sideBySide}>
+        <div className={styles.inputFieldContainer}>
+          <label htmlFor="name">Name</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            className={styles.inputHeight}
+            defaultValue={name}
+            onChange={(e) => setName(e.target.value)}
+            required
+          />
+        </div>
+        <div className={styles.inputFieldContainer}>
+          <label htmlFor="tel">Phone</label>
+          <input
+            type="tel"
+            name="tel"
+            id="tel"
+            className={styles.inputHeight}
+            defaultValue={phone}
+            onChange={(e) => setPhone(e.target.value)}
+            required
+          />
+        </div>
+      </div>
+      <div className={styles.inputFieldContainer}>
+        <label htmlFor="email">Email</label>
+        <input
+          type="email"
+          name="email"
+          id="email"
+          className={styles.inputHeight}
+          defaultValue={email}
+          onChange={(e) => setEmail(e.target.value)}
+          required
+        />
+      </div>
+      {ClientFormDataDropdowns.map((item, index) => {
+        return <Dropdown item={item} key={index} changeEvent={handleChange} />;
+      })}
+
+      <div
+        className={injuries === "Yes" ? styles.textAreaContainer : styles.hide}
+      >
+        <label htmlFor="yesInjuries">Please describe your injury</label>
+        <textarea
+          name="yesInjuries"
+          id="yesInjuryes"
+          className={styles.textarea}
+          defaultValue={yesInjuries}
+          onChange={(e) => setYesInjuries(e.target.value)}
+          required={injuries === "Yes"}
```

Comment:
itd be best to create a types file and create enums for the question ids and answers. that way you dont have multiple arbitrary checks for values like this and if that value changes, you dont have to update all your usages